.travis.yml

```diff
@@ -2,24 +2,33 @@ language: ruby
 cache: bundler
 
 rvm:
-  - 2.3.8
-  - 2.4.6
-  - 2.5.5
-  - 2.6.3
+  - 2.3
+  - 2.4
```

Comment:
Do we still need to support `2.3` and `2.4`?

Reply:
talked with @jfeltesse-mdsol and probably we should drop them with the next major release (`v7.0`)